sqlalchemy_exasol/version.py

```diff
@@ -0,0 +1,9 @@
+# ATTENTION: 
+# This file is generated, do not edit this file manually!
+# If you need to change the version, do so in the project.toml, e.g. by using `poetry version X.Y.Z`.
```

Comment:
Regarding the upgrade to python 3.8, I think this project can wait longer with that, we only need to drop 3.6 if didn't do it yet. And, please don't mix things in issues together. The version file is a separate issue. I guess you want to use importlib.metadata, for pre 3.8 there is a Backport package, you can use. We already used it already various other projects.

Reply:
However, there is another issue with importlin_metadata. From what we saw, it only works for installed packages.

Reply:
@tkilias that it works only on installed package is fine in the scenario of the sqlaclhemy plugin I would say (I don't expect the extension points to work without installation, I am not 100% sure though).

Regarding `importlib.metadata` previous to 3.8 where it is part of the standard library can be hell of a mess if more than one library uses it, which is often the case (We don't want to run into dependency issues because of the version imho).